coding-the-legislation/reforms.md

```diff
@@ -50,8 +50,9 @@ def build_reform(tax_benefit_system):
     class charges_deduc(formulas.SimpleFormulaColumn):
         label = u"Charge déductibles always returning 999"
         reference = charges_deductibles.charges_deduc
+        definition_period = YEAR
         def function(self, simulation, period):
```

Comment:
Inconsistent whitespace in example: there was always a blank line between class variables and function definition.